### PR TITLE
first pass at handling multiple errors

### PR DESCRIPTION
**Pull request by tkellen**
Attempts closing #608.  WIP.

### CODE REVIEW

File: format/index.md [hunk 1]
```diff
@@ -855,6 +855,18 @@ A server **MUST** prepare responses, and a client **MUST** interpret
 responses, in accordance with
 [`HTTP semantics`](http://tools.ietf.org/html/rfc7231).
 
+### Errors
+
+A server **MAY** choose to stop processing operations as soon a problem is
```
Comment (dgeb) [inline format/index.md]: ... as soon _as_ a problem is ...

File: format/index.md [hunk 2]
```diff
@@ -855,6 +855,18 @@ A server **MUST** prepare responses, and a client **MUST** interpret
...
+encountered, or it **MAY** continue processing operations and encounter
```
Comment (dgeb) [inline format/index.md]: perhaps remove "operations" here - that really just applies to jsonpatch operations

File: format/index.md (hunk 1 above)
Comment (tkellen) [inline format/index.md]: fixed

File: format/index.md (hunk 2 above)
Comment (tkellen) [inline format/index.md]: fixed

**Pull request merged**